### ISSUE TITLE
"~/.config/itch/prereqs/firejail-386 missing" when launching windows games on linux

### ISSUE THREAD

**Issue by julikaiba** [itchio/itch#2732]
Whenever I try to launch a windows game using the Itch launcher i get the error "fork/exec /home/j/.config/itch/prereqs/firejail-386/firejail: no such file or directory".
When i checked the folder prereqs only contains "firejail-amd64" "info.json" and "windows-386".

I installed the launcher from the AUR on manjaro.

**Comment by centuryglass** [comment]
This is the workaround I used for this problem.  Not sure if this is the best approach, but it seems to work.
```
sudo pacman -Su firejail
mkdir ~/.config/itch/prereqs/firejail-386
ln -s /usr/bin/firejail ~/.config/itch/prereqs/firejail-386/firejail
```

**Comment by neogeographica** [comment]
Yeah this is puzzling. I ran into this launching Cloud Gardens on an Ubuntu-variant OS (elementary OS 5). I submitted the log through the error report mechanism... haven't tried any workaround yet. Curious if there's more detail about what it's trying to find here; does it expect some different firejail executable from the one under firejail-amd64?

**Comment by theMarix** [comment]
I am seeing the same issue on openSUSE.

**Comment by TheLastProject** [comment]
Can confirm this is still happening also under Fedora 37. The workaround described by @centuryglass works, but this seems like something itch.io should set up properly itself.

**Comment by GeoffreyCoulaud** [comment]
Can confirm this is still happening, I'm using EndeavourOS and the itch launcher is installed via flatpak.

Edit:
The problem comes from the itch sandbox that is using `firejail` on linux.   
If you disable it in the app's settings, the problem is gone.

**Comment by michaelneverwins** [comment]
I think the reason for this error is described here: https://github.com/itchio/itch/issues/2457#issuecomment-653865910
>it's looking for a 32-bit firejail (but it should be looking at the host's architecture when Wine is used, not the target architecture)

Installing `firejail` at the system level and linking to it as suggested by @centuryglass

>```
>mkdir ~/.config/itch/prereqs/firejail-386
>ln -s /usr/bin/firejail ~/.config/itch/prereqs/firejail-386/firejail
>```

may be the best solution, especially considering the version of `firejail` that the app installs is probably older than what most up-to-date Linux distributions provide. For the same reason, one may also wish to link `~/.config/itch/prereqs/firejail-amd64/firejail` to `/usr/bin/firejail` as described here: https://github.com/itchio/itch/issues/1150#issuecomment-1716534334

However, if one wishes to let the app manage its own copy of `firejail` without having a package manager install it to `/usr/bin/`, just linking `~/.config/itch/prereqs/firejail-386/firejail` to `~/.config/itch/prereqs/firejail-amd64/firejail` will probably work.
```bash
mkdir ~/.config/itch/prereqs/firejail-386
ln -s ~/.config/itch/prereqs/firejail-amd64/firejail ~/.config/itch/prereqs/firejail-386/firejail
```

**Comment by leafo** [comment]
The app should prefer system firejail now, and the platform arch detection should be working correctly for installing local firejail.
